<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderElement</class>
 <widget class="QWidget" name="OrderElement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: black;
	background-color: white;
	border-top: none;
	border-bottom: 1px dashed rgb(18, 18, 23);
	border-left: none;
	border-right: none;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="productName_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
padding-left: 20px;
padding-right: 20px;</string>
     </property>
     <property name="text">
      <string>Pizza</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="productPrice_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
padding-left: 10px;</string>
     </property>
     <property name="text">
      <string>2500</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="quantity_layout">
     <item>
      <widget class="QPushButton" name="reduceQuantity_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(0, 153, 73);
  border: none;
  color: white;
  text-align: center;
 padding-bottom: 5px;
}

QPushButton:hover {
  background-color: rgb(218, 213, 207);     /* Fondo al pasar el mouse */
  color: white;             /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: transparent;     /* Fondo al presionar */
                /* Color del texto al presionar */
	color: rgb(52, 52, 52);
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="quantity_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="incrementQuantity_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(0, 153, 73);
  border: none;
  color: white;
  text-align: center;
 padding-bottom: 5px;
}

QPushButton:hover {
  background-color: rgb(218, 213, 207);     /* Fondo al pasar el mouse */
  color: white;             /* Color del texto al pasar el mouse */
}

QPushButton:pressed {
  background-color: transparent;     /* Fondo al presionar */
                /* Color del texto al presionar */
	color: rgb(52, 52, 52);
}

QPushButton:disabled {
  background-color: rgb(39, 39, 43);     /* Fondo deshabilitado */
  color: #7f8c8d;                /* Color del texto deshabilitado */
}

QPushButton:checked {
  background-color: #27ae60;     /* Fondo cuando está checkable y activado */
  color: #ecf0f1;        
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="totalPrice_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
padding-left: 15px;</string>
     </property>
     <property name="text">
      <string>2500</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
